### DISCUSSION TITLE
Cursor lag mod

### DISCUSSION THREAD

**Discussion by MightyHelper** [ppy/osu#13083] [Ideas] [answered]
**Describe the new feature:** 
The idea came from writing #6894
The cursor on screen shows the location of the real cursor some time before.
Example:
Real cursor (100x,100y)
   Move over 1s to (200x,200y)
If lag time is 1s when the real cursor gets to (200x,200y) the on-screen cursor would start to move.
Note(Clicking would click where the real cursor is).
**Proposal designs of the feature:** 
Add multiple stages to cycle through on lag time (0.1s, 0.2s, 0.3s, 0.4s, 0.5s, 1s, 2s).
Maybe adjust for #6272?
Or maybe scale with AR?

**Comment by Phippe** [comment]
I feel like this is gonna mess pretty badly with your muscle memory or could be frustrating to play at least. You always try to eliminate any lag, so deliberatly introducing it seems kinda wrong ^^

**Comment by mk56-spn** [comment, marked as answer]
will be closed by : https://github.com/ppy/osu/pull/21673
